daemons/fenced/fenced_commands.c

```diff
@@ -2921,10 +2922,12 @@ handle_request(pcmk__client_t *client, uint32_t id, uint32_t flags,
         pcmk__ipc_send_xml(client, id, reply, flags);
         client->request_id = 0;
         free_xml(reply);
-        return 0;
+        rc = pcmk_ok;
```

Comment:
if I didn't oversee anything reply is just used in this block.
but still - to prevent future mistakes - we might clear it here after free.
or simply pull reply entirely into this block.

Reply:
Yep, you probably saw that was fixed later, in ee0cfb6b2 :)